test/docker/compose.go

```diff
@@ -446,16 +442,27 @@ func (d *Compose) Start(ctx context.Context) (*DockerCompose, error) {
 		}
 		containers = append(containers, container)
 
-		if d.withMinIOBucketName != "" {
-			if err := createBucket(ctx, container.URI(), "us-west-2", d.withMinIOBucketName); err != nil {
-				return nil, err
+		if len(d.withBackendS3Buckets) > 0 {
+			for _, bName := range d.withBackendS3Buckets {
+				if bName == "" {
+					continue
+				}
+				if err := createBucket(ctx, container.URI(), "eu-west-1", bName); err != nil {
```

Comment:
I would move the `createBucket` method into `minio.go` and pass `withBackendS3Buckets` to the `MiniIO` container and create buckets there inside like we do this in [ollama](https://github.com/weaviate/weaviate/blob/main/test/docker/ollama.go#L58-L68) container here just after the ollama container starts we pull the models to it, then that prepared container is returned to compose.go.

You can run the `createBucket` operation [here](https://github.com/weaviate/weaviate/blob/main/test/docker/minio.go#L54) just after the `MinIO` container starts, so that it will be returned to compose.go as already prepared container.

Reply:
good idea!